basyx/aas/adapter/xml/xml_deserialization.py

```diff
@@ -85,8 +85,9 @@ def _tag_replace_namespace(tag: str, nsmap: Dict[str, str]) -> str:
     """
     split = tag.split("}")
     for prefix, namespace in nsmap.items():
-        if namespace == split[0][1:]:
-            return prefix + ":" + split[1]
+        if prefix:
+            if namespace == split[0][1:]:
```

Comment:
We don't need nesting here, instead please combine both conditions into one using an `and`.